documentation/antora-playbook.yml

```
@@ -0,0 +1,16 @@
+site:
+  title: Antora Demo Site
+  # the 404 page and sitemap files only get generated when the url property is set
+  url: https://dqualizer.github.io/dqualizer
+  start_page: dqualizer-main::landingpage.adoc
+content:
+  sources:
+    - url: https://github.com/dqualizer/dqualizer.git
+      branches: docs
+      start_path: docu-stuff/docs
```

I think this should be "documentation".

```suggestion
      start_path: documentation/docs
```